<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC7MenuWidget</class>
 <widget class="QWidget" name="LC7MenuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>116</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QPushButton" name="saveSessionAsButton">
     <property name="text">
      <string>Save Session As...</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+Shift+S</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="openSessionButton">
     <property name="text">
      <string>Open Session</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+O</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="saveSessionButton">
     <property name="text">
      <string>Save Session</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+S</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="newSessionButton">
     <property name="text">
      <string>New Session</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+N</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="quitButton">
     <property name="text">
      <string>Quit</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+Q</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
